pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -147,13 +157,14 @@ func NewManager(cpuPolicyName string, reconcilePeriod time.Duration, machineInfo
 		machineInfo:                machineInfo,
 		nodeAllocatableReservation: nodeAllocatableReservation,
 	}
+	manager.sourcesReady = &sourcesReadyStub{}
```

Comment:
Why add this line when constructing manager?

Reply:
initializes sourcesReady to perform no-op operations.

Reply:
I mean why not add this line after [L163](https://github.com/kubernetes/kubernetes/pull/81344/files#diff-67c46e7cbb41ea5e675000a3298e6b32R163)